File: demos/jans-chip/android/app/src/main/java/io/jans/chip/model/fido/assertion/result/AssertionResultRequest.kt
package io.jans.chip.model.fido.assertion.result

import androidx.room.Ignore

class AssertionResultRequest {
     var id: String? = null

     var type: String? = null

     var rawId: String? = null

     var response: Response? = null

     @Ignore
     var isSuccessful: Boolean? = true

     @Ignore
     var errorMessage: String? = null
}

File: demos/jans-chip/ios/Jans-Chip/Models/Networking/AssertionResultRequest.swift
//
//  AssertionResultRequest.swift
//  Jans-Chip
//
//  Created by Nazar Yavornytskyi on 25.07.2024.
//

import Foundation

struct AssertionResultRequest: Codable, ErrorHandler {
    
    var errorMessage: String?
    var isSuccess: Bool = true
    
    private enum CodingKeys: String, CodingKey {
        case id
        case type
        case rawId
    }
    
    var id: String?
    var type: String?
    var rawId: String?
    var response: Response?
}
